packages/feedback-modal/README.md

```diff
@@ -0,0 +1,25 @@
+<p align="center">
+  <a href="https://sentry.io/?utm_source=github&utm_medium=logo" target="_blank">
+    <img src="https://sentry-brand.storage.googleapis.com/sentry-wordmark-dark-280x84.png" alt="Sentry" width="280" height="84">
+  </a>
+</p>
+
+# Sentry Integration for Feedback
```

Comment:
do we want to update the README for feedback-modal and feedback-screenshot?

Reply:
Yeah maybe point to feedback README?